packages/common/pipes/src/includes/includes.pipe.ts

```diff
@@ -0,0 +1,8 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({ name: 'includes' })
+export class IncludesPipe implements PipeTransform {
+    public transform<T>(array: T[] | null | undefined, entry: T): boolean {
+        return array?.includes(entry) ?? false;
```

Comment:
Maybe we can create util `includes` from `@angular-ru/common/array`

```ts
public transform<T>(array: T[] | null | undefined, entry: IncludeElement<T>): boolean {
        return includes(array, entry);
}
```

```ts
export type IncludeEntry<T> =  T | T[] | RegExp;

export function includes<T>(T[] | null | undefined, entry: IncludeEntry<T>): boolean {
  // what do you think to improve the ability to search across multiple parameters?
}
```

Reply:
i think it's all — a different functions.
`includesAny`, `includesAll`, `includesMatching`

Reply:
good point, do you want implements it’s?

Reply:
not here. moreover i think to use this functions as a pipe will complicate a template and you should find another way to calculate it. in short i'm against to make pipes from this

Reply:
sure, create issue about your proposal